trl/trainer/sft_config.py

```diff
@@ -57,6 +57,8 @@ class SFTConfig(TrainingArguments):
             If `None`, no truncation is applied. When packing is enabled, this value sets the sequence length.
         packing (`bool`, *optional*, defaults to `False`):
             Whether to pack multiple sequences into a fixed-length format. Uses `max_length` to define sequence length.
+        packing_strategy (`str`, *optional*, defaults to `"ffd"`):
+            Strategy for packing sequences. Can be either `"ffd"` (first-fit decreasing, default), or `"fixed"`.
```

Comment:
`fixed` seems ambiguous? maybe 'legacy' or 'non-contiguous' can be better?
And add a note here that old method slices sequences across elements which adds cross-document contamination.

Reply:
`streamed`?